queries.sql

```diff
@@ -79,47 +79,87 @@
 -- SELECT name, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY name;
 
 -- What animals belong to Melody Pond?
-SELECT animals.name
-FROM animals
-JOIN owners ON animals.owner_id = owners.id
-WHERE owners.full_name = 'Melody Pond';
-
--- List of all animals that are pokemon (their type is Pokemon).
-SELECT animals.name
-FROM animals
-JOIN species ON animals.species_id = species.id
-WHERE species.name = 'Pokemon';
-
--- List all owners and their animals, remember to include those that don’t own any animal.
-SELECT owners.full_name, animals.name
-FROM owners
-LEFT JOIN animals ON owners.id = animals.owner_id;
-
--- How many animals are there per species?
-SELECT species.name, COUNT(animals.id) AS count
-FROM species
-LEFT JOIN animals ON species.id = animals.species_id
-GROUP BY species.name;
-
--- List all Digimon owned by Jennifer Orwell.
-SELECT animals.name
-FROM animals
-JOIN species ON animals.species_id = species.id
-JOIN owners ON animals.owner_id = owners.id
-WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
-
--- List all animals owned by Dean Winchester that haven’t tried to escape.
-SELECT *
-FROM animals
-JOIN owners ON animals.owner_id = owners.id
-WHERE owners.full_name = 'Dean Winchester' AND escaped IS NULL;
-
--- Who owns the most animals?
-SELECT owners.full_name, COUNT(animals.id) AS count
-FROM owners
-LEFT JOIN animals ON owners.id = animals.owner_id
-GROUP BY owners.full_name
-ORDER BY count DESC LIMIT 1;
+-- SELECT animals.name
+-- FROM animals
+-- JOIN owners ON animals.owner_id = owners.id
+-- WHERE owners.full_name = 'Melody Pond';
+
+-- -- List of all animals that are pokemon (their type is Pokemon).
+-- SELECT animals.name
+-- FROM animals
+-- JOIN species ON animals.species_id = species.id
+-- WHERE species.name = 'Pokemon';
+
+-- -- List all owners and their animals, remember to include those that don’t own any animal.
+-- SELECT owners.full_name, animals.name
+-- FROM owners
+-- LEFT JOIN animals ON owners.id = animals.owner_id;
+
+-- -- How many animals are there per species?
+-- SELECT species.name, COUNT(animals.id) AS count
+-- FROM species
+-- LEFT JOIN animals ON species.id = animals.species_id
+-- GROUP BY species.name;
+
+-- -- List all Digimon owned by Jennifer Orwell.
+-- SELECT animals.name
+-- FROM animals
+-- JOIN species ON animals.species_id = species.id
+-- JOIN owners ON animals.owner_id = owners.id
+-- WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
+
+-- -- List all animals owned by Dean Winchester that haven’t tried to escape.
+-- SELECT *
+-- FROM animals
+-- JOIN owners ON animals.owner_id = owners.id
+-- WHERE owners.full_name = 'Dean Winchester' AND escaped IS NULL;
+
+-- -- Who owns the most animals?
+-- SELECT owners.full_name, COUNT(animals.id) AS count
+-- FROM owners
+-- LEFT JOIN animals ON owners.id = animals.owner_id
+-- GROUP BY owners.full_name
+-- ORDER BY count DESC LIMIT 1;
+
+
+
```

Comment:
- [ ] Please remove unused code so that your code base is clean and uncluttered.